marathon/marathon_stub.go

```diff
@@ -4,46 +4,56 @@ import (
 	"errors"
 
 	"github.com/allegro/marathon-consul/apps"
+	"sync"
 )
 
 type MarathonerStub struct {
-	AppsStub     []*apps.App
-	AppStub      map[apps.AppID]*apps.App
-	TasksStub    map[apps.AppID][]*apps.Task
-	leader       string
-	interactions bool
+	AppsStub       []*apps.App
+	AppStub        map[apps.AppID]*apps.App
+	TasksStub      map[apps.AppID][]*apps.Task
+	leader         string
+	interactionsMu sync.RWMutex
+	interactions   bool
```

Comment:
I don't think it's our case. We don't target to ARM or x86 and we only need `uint32`